pkg/ip/link_linux.go

```diff
@@ -60,11 +60,15 @@ func peerExists(name string) bool {
 	return true
 }
 
-func makeVeth(name string, mtu int) (peerName string, veth netlink.Link, err error) {
+func makeVeth(name, vethPeerName string, mtu int) (peerName string, veth netlink.Link, err error) {
 	for i := 0; i < 10; i++ {
```

Comment:
If `vethPeerName` is specified, the retry here will no longer be needed.

Reply:
yeah, you are right. I have made it skip the retry if the vethPeerName is set.